frontend/src/hooks/useLoading.ts

```diff
@@ -16,7 +16,7 @@ const useLoading = (loading: boolean, selector = '[data-loading=true]') => {
                 }
             });
         }
-    }, [loading, ref]);
+    }, [loading]);
```

Comment:
This was the cause, with ref as dependency, it was over reactive and overwriting skeleton, when multiple components were using useLoading.